examples/cassandra/README.md

```diff
@@ -305,6 +305,97 @@ UN  10.244.0.5  74.09 KB   256     49.7%             86feda0f-f070-4a5b-bda1-2ee
 UN  10.244.3.3  51.28 KB   256     51.0%             dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
 ```
 
+### Using a DaemonSet
+
+In Kubernetes a _[Daemon Set](../../docs/admin/daemons.md)_ can distribute pods onto Kubernetes nodes, one-to-one.  Like a _ReplicationController_ it has a selector query which identifies the members of it's set.  Unlike a _ReplicationController_ it has a node selector to limit which nodes are scheduled with the templated pods, and replicates not based on a set target number of pods, but rather assigns a single pod to each targeted node.
+
+An example use case: when deploying to the cloud, the expectation is that instances are ephemeral and might die at any time. Cassandra is built to replicate data across the cluster to facilitate data redundancy, so that in the case that an instance dies, the data stored on the instance does not, and the cluster can react by re-replicating the data to other running nodes.
+
+DaemonSet is designed to place a single pod on each node in the Kubernetes cluster. If you're looking for data redundancy with Cassandra, let's create a daemonset to start our storage cluster:
```

Comment:
The case where I would use this is if I was using cassandra on bare metal, and I was storing the data in a hostDir instead of an emptyDir, and I wanted to ensure that a cassandra daemon started on all nodes (or all matching nodes), and that it used any existing files in /var/lib/cassandra from the previous pod, thus saving the network cost of reconstruction, when perfectly good data is still there.  This allows you to have less downtime due to reconstruction after a node reboot/power-down.  If you want to change it to use a hostDir, and can verify that a cassandra node will come back after reboot without reconstruction, then I will be happy to take this PR.  If you don't have access to bare metal, simulating with PD is fine too.